AK/stdarg.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+typedef __builtin_va_list va_list;
```

Comment:
Perhaps we should have a note at the top of this file that it must remain includable by C files, since it's used as part of LibC. 

In the future we could maybe remove the printf-like interface from the kernel and only use AK/Format.h instead. I think this is a good follow-up migration issue to create